modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -179,6 +179,7 @@ export abstract class BaseCoin implements IBaseCoin {
    * to big units (btc, eth, xrp, xlm)
    */
   baseUnitsToBigUnits(baseUnits: string | number): string {
+    BigNumber.set({ DECIMAL_PLACES: 24 });
```

Comment:
if the coin needs 24 decimal places it will use 24, but if it does not, the coin willuse the number of decimal places it needs, it will not round the value